build_deps/configure.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2022 The KerasCV Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+# Usage: python configure.py
+#
+
+
+import logging
+import os
+import pathlib
+import platform
+
+import tensorflow as tf
+from packaging.version import Version
+
+_TFA_BAZELRC = ".bazelrc"
+
+
+# Writes variables to bazelrc file
+def write(line):
```

Comment:
Same question with build_pip_pkg.sh

Reply:
Added a little doc blurb at the top -- is there anything else specifically you're looking for here?